src/App.tsx

```diff
@@ -15,48 +15,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NAME = 'name',
+  LENGTH = 'length',
+}
+
+function sortArray(
+  goods: string[],
+  howSort: SortType,
+  isReversed: boolean,
+) {
+  const array = [...goods];
+
+  array.sort((value1, value2) => {
+    if (howSort === SortType.NAME) {
+      return value1.localeCompare(value2);
+    }
+
+    if (howSort === SortType.LENGTH) {
+      return value1.length - value2.length;
+    }
```

Comment:
You can use the switch/case instead. Let's imagine that there will be more than 2 options